tool.go

```diff
@@ -143,7 +143,15 @@ func main() {
 					}
 					if len(pkgs) == 1 { // Only consider writing output if single package specified.
 						if pkgObj == "" {
-							pkgObj = filepath.Base(pkg.ImportPath) + ".js"
+							if pkg.ImportPath == "." {
+								dirPath, err := os.Getwd()
```

Comment:
No need to do `os.Getwd` here, you can use [`currentDirectory`](https://github.com/gopherjs/gopherjs/blob/b40cd48c38f9a18eb3db20d163bad78de12cf0b7/tool.go#L42) variable:

https://github.com/gopherjs/gopherjs/blob/b40cd48c38f9a18eb3db20d163bad78de12cf0b7/tool.go#L42-L50